packages/paste-core/components/form-pill-group/stories/customization.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
```

Comment:
I like moving Customization stories into their own sub-story folder and file because it keeps things clean in the storybook sidebar. Hence this change.